docker-compose.yml

```diff
@@ -181,3 +181,12 @@ services:
       - ALLOW_EMPTY_PASSWORD=yes
     ports:
       - 6379:6379
+
+  redoc:
+    container_name: instill-doc
+    image: volbrene/redoc:1.2.0
+    environment:
+      - 'URLS=[{url: ''https://raw.githubusercontent.com/instill-ai/protobufs/main/pipeline/pipeline.swagger.json'', name: ''Pipeline''},{url: ''https://raw.githubusercontent.com/instill-ai/protobufs/main/model/model.swagger.json'', name: ''Model''}]'
+      - PAGE_TITLE=Instill Doc
```

Comment:
```suggestion
    container_name: openapi
    image: volbrene/redoc:1.2.0
    environment:
      - 'URLS=[{url: ''https://raw.githubusercontent.com/instill-ai/protobufs/main/pipeline/pipeline.swagger.json'', name: ''Pipeline''},{url: ''https://raw.githubusercontent.com/instill-ai/protobufs/main/model/model.swagger.json'', name: ''Model''}]'
      - PAGE_TITLE=Instill AI - API Reference
```

Reply:
Also, the service is not added to the make file.

Reply:
> Ok, this PR is dependent on the merge of [instill-ai/protobufs#14](https://github.com/instill-ai/protobufs/pull/14).

Yeah, I should add dependency of issues as well

Reply:
> Also, the service is not added to the make file.

Thanks for the correction. It is fixed in this [commit](https://github.com/instill-ai/vdp/pull/48/commits/d1123a7cf4e23f4da6bc217dc6057c0bf219ff57)

Reply:
This [commit](https://github.com/instill-ai/vdp/pull/48/commits/397aa2648ee281b0116b8500f533855b5a557a9b) refactors suggestions of `docker-compose.yml`

Reply:
@xiaofei-du l leave the `Resolve conversation` button for you. Please help to check again